docs/ops/arithmetic/Tanh_1.md

```diff
@@ -23,3 +29,24 @@ element in the output tensor with the following formula:
 \f[
 tanh ( x ) = \frac{2}{1+e^{-2x}} - 1 = 2sigmoid(2x) - 1
 \f]
```

Comment:
Please, describe how it will work for integer numbers. How the rounding is performed?

Reply:
👍